echopype/tests/convert/test_convert_azfp.py

```diff
@@ -196,7 +196,7 @@ def test_convert_azfp_01a_pressure_temperature(azfp_path):
 
 def test_load_parse_azfp_xml(azfp_path):
     azfp_xml_path = azfp_path / '23081211.XML'
-    parseAZFP = ParseAZFP(None, str(azfp_xml_path))
+    parseAZFP = ParseAZFP(None, str(azfp_xml_path), {}, "")
```

Comment:
Seems like this line is ok without the additions?

Reply:
Ah yeah I removed this. Thanks!